tree/ntupleutil/v7/inc/ROOT/RNTupleInspector.hxx

```diff
@@ -169,6 +172,20 @@ public:
    /// Get the IDs of all columns with the given type.
    const std::vector<DescriptorId_t> GetColumnsByType(EColumnType);
 
+   /////////////////////////////////////////////////////////////////////////////
+   /// \brief Get a histogram showing information for each column type present,
+   ///
+   /// \param[in] histKind Which type of information should be returned.
+   /// \param[in] histName The name of the histogram. An empty string means a default name will be used.
+   /// \param[in] histTitle The title of the histogram. An empty string means a default title will be used.
+   ///
+   /// \return A pointer to a `TH1D` containing the specified kind of information.
+   ///
+   /// Get a histogram showing the count, number of elements, size on disk, or size in memory for each column
+   /// type present in the inspected RNTuple.
```

Comment:
N.B. I will change the documentation for the other methods to this format in another PR.